source/code/providers/Container_DaemonEvent_Class_Provider.cpp

```diff
@@ -18,6 +18,7 @@
 #include "Container_ContainerLogFileReader.h"
 
 #define LASTQUERYTIMEFILE "/var/opt/microsoft/docker-cimprov/state/LastEventQueryTime.txt"
+#define TEST_LASTQUERYTIMEFILE "/tmp/LastEventQueryTime.txt"

```

Comment:
Perhaps so. But I wouldn't have done this, and I think it's a poor example. What else did you see that did that? Was it in the docker provider or elsewhere?

Just because you see some yucky code doesn't mean it should be duplicated! :smile: